Support.md

```diff
@@ -81,3 +81,30 @@ The Senior Support Engineer role builds upon the foundation of the Support Engin
 - I am actively introspective and take time to work on my personal and professional development.
 - I happily take responsibility for my own mistakes and I learn from the mistakes of others.
 
+## Lead Support Engineer
+
+The Lead Support Engineer role builds upon the foundation of the Support Engineer and Senior Support Engineer roles.
+
+> I am a technical leader, a person that is recognised both within the Support team and as part of the wider Octopus community for the breadth and depth of your technical ability.
+
+### I am trusted
+
+- I have mastered my tools and the core building blocks used by our team.
+- I have a a deep technical understanding of more than one area of Octopus, and can confidently use this knowledge to assist customers and fellow support team members in all areas of Octopus.
+- I know when it makes sense to push back against a customer, and when it is appropriate to not create an issue
+- I know how to mentor a customer to the best path, and understand that what they are asking for may not be what they need
+
+
+### I am a mentor of mentors
+
+- I know when to assist with a ticket, and when to push to encourage others to learn more.
+- I help others in my team to become mentors.
+- I am always on the lookout for opportunities to teach and show others alternate, cleaner ways to delight our customers in an ego-less manner. 
+- I am working to improve our processes, making it easier for everyone in our team to grow and learn.
+
+
+### Everything is a learning oppurtunity
+
```

Comment:
This might be covered in the other points, but I think it's worth calling out for a lead position:
- I share the lessons I learn so the entire team and Octopus can benefit

Reply:
That's part of the Senior Role:
- I can effectively coach people and pass on my knowledge.
- I am a confident and proficient teacher of my craft.

But I'm going to add that specific line again anyway :)